artifact-management/commands/gem-test-multiple-ruby-versions.yml

```diff
@@ -0,0 +1,28 @@
+description: Perform gem test with many versions of Ruby
+parameters:
+  gem_test_dir:
+    description: The directory in which the gem tesets should run
+    type: string
+    default: .
+  test_command:
+     default: bundle exec rake test
+     description: The command that should be used to run tests
+     type: string
+  ruby_versions:
+    description: The ruby versions to be used to test the gem.
+    type: string
+    default: "2.2.10,2.3.8,2.4.10,2.5.8,2.6.6,2.7.1"
```

Comment:
Although I think it's great that our `avvo/ruby-testing` Docker image has Ruby 2.7.x, by default we probably don't want to test against it unless explicitly requested. Why?

- Ruby 2.7.x uses Bundler 2.0, which may clash with the specified Bundler version in the `Gemfile.lock` file.
- Ruby 2.7.x (and even 2.6.x) include some new behaviors that are likely to conflict with older Ruby code. In other words, the tests probably won't pass in most cases.